src/main/java/core/basesyntax/transactionprocessorimpl/FruitTransactionProcessorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.transactionprocessorimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.transactionprocessor.FruitTransactionProcessor;
+import core.basesyntax.transactionprocessor.OperationHandler;
+import core.basesyntax.transactionprocessor.OperationStrategy;
+import java.util.List;
+
+public class FruitTransactionProcessorImpl implements FruitTransactionProcessor {
+    private static final OperationStrategy STRATEGY = new OperationStrategyImpl();

```

Comment:
it shouldn't be constant, just a regular global variable and you should initialize it in the constructor